src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -173,25 +174,26 @@ public void handleHelp() {
      * Opens file manager for import window
      */
     @FXML
-    public void handleImport () throws ParseException, CommandException {
-        JFileChooser fileChooser = new JFileChooser();
-        JDialog dialog = new JDialog();
-        fileChooser.setCurrentDirectory(new File(System.getProperty("user.dir")));
-        int returnValue = fileChooser.showOpenDialog(dialog);
-        if (returnValue == JFileChooser.APPROVE_OPTION) {
-            File selectedFile = fileChooser.getSelectedFile();
-            ImportFileParser converter = new ImportFileParser();
-            List<String> res;
+    public void handleImport () throws CommandException, ParseException {
+        FileChooser fileChooser = new FileChooser();
+        Stage stage = new Stage();
+        fileChooser.setInitialDirectory(new File(System.getProperty("user.dir")));
+        File selectedFile = fileChooser.showOpenDialog(stage);
+        String fileType = selectedFile.toString();
+        if (!Files.getFileExtension(fileType).equals("xlsx") && !Files.getFileExtension(fileType).equals("xls")) {
```

Comment:
`if (!Files.getFileExtension(fileType).equals("xlsx") || !Files.getFileExtension(fileType).equals("xls"))`

Shouldnt this be a logical disjunction instead since a xlsx is also a xls file but a xls file is not an xlsx file?

I've changed the xlsx extension to xls extension and it does not work even though it is an excel file.